feat: add `$app/state` module (#13140)
sveltejs/kit: documentation/docs/30-advanced/25-errors.md

}
```

This throws an exception that SvelteKit catches, causing it to set the response status code to 404 and render an [`+error.svelte`](routing#error) component, where `page.error` is the object provided as the second argument to `error(...)`.

```svelte
<!--- file: src/routes/+error.svelte --->
<script>
	import { page } from '$app/state';
</script>

<h1>{page.error.message}</h1>
```

> [!LEGACY]

> > `$app/state` exists since SvelteKit 2.12. If you're using an earlier version or are using Svelte 4, use `$app/stores` instead.

Author: ```suggestion
> `$app/state` was added in SvelteKit 2.12. If you're using an earlier version or are using Svelte 4, use `$app/stores` instead.
```
